helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -1548,6 +1547,45 @@
         }
       ]
     },
+    {
+      "if": {
+        "properties": {
+          "distribution": { "pattern": "^[^g].[^k]*[^e].*" }
+        },
+        "required": ["distribution"]
+      },
+      "then": {
+        "if": {
+          "properties": {
+            "distribution": { "pattern": "^[^e].[^k]*[^s].*" }
```

Comment:
Is it possible to simplify validation logic for new folks? I was thinking of combining the 2 value checks into 1 with a value like this.
`"distribution": { "pattern": "^(gke|eks).*" }`

Reply:
Good call, updated.